.github/workflows/link-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Links
+
+on:
+  push:
+    branches:
+      - main
+      - kc/link-checker
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "00 18 * * *"  # This will run the check daily at 18:00 UTC
```

Comment:
Let's schedule this to run when someone opens a PR, not dailly.

Reply:
done